src/incubator/TextField/FloatingPlaceholder.tsx

```diff
@@ -18,24 +19,29 @@ const FloatingPlaceholder = ({
   floatOnFocus,
   validationMessagePosition,
   extraOffset = 0,
+  defaultValue,
   testID
 }: FloatingPlaceholderProps) => {
   const context = useContext(FieldContext);
   const [placeholderOffset, setPlaceholderOffset] = useState({
     top: 0,
     left: 0
   });
-  const animation = useRef(new Animated.Value(Number((floatOnFocus && context.isFocused) || context.hasValue))).current;
+
+  const shouldFloat = useMemo(() => {
```

Comment:
There's no need to memoize primitive values (booleans, strings, etc...) 😬

Reply:
Wouldn't that break the `useDidUpdate` dependencies? Unless I put the dependencies from here, which is a little weird.

Reply:
I think you're right but I found a bug :/